examples/lifelong_learning/atcii/README.md

```diff
@@ -16,15 +16,15 @@ In this example, you can use [ASHRAE Global Thermal Comfort Database II](https:/
 
 
 
-download [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), including train, evaluation and incremental dataset.
+We provide a well-processed [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), which including train, evaluation and incremental dataset.
```

Comment:
So there are three types of dataset:
train/evaluation/incremental 

I download and extract it, and three files are extracted:
```
testData.csv
trainData2.csv
trainData.csv
```
So which is train, which is evalution, which is incremental?